.github/workflows/chart-preview.yml

```
@@ -117,8 +123,28 @@ jobs:
 
+      - id: build-plane-mcp-server
+        if: ${{ env.BUILD_PLANE_MCP_SERVER == 'true' }}
+        name: Build Plane-MCP-Server
+        working-directory: code
+        env:
+          EXPORT_DIR: ${{env.PREVIEW_BUILD_FOLDER}}
+          CHART_REPO: plane-mcp-server
+          CR_KEY: ${{ env.GPG_KEY_NAME }}
+          CR_PASSPHRASE_FILE: ${{env.GNUPGHOME}}/gpg-passphrase
+          CR_KEYRING: ${{env.GNUPGHOME}}/secring.gpg
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:6:106: Double quote to prevent globbing and word splitting

(shellcheck)